doc/Advanced.md

```diff
@@ -1310,7 +1310,7 @@ The new syntax is
 ```
 It allows to have variations of the parameters depending on the run and beam types. The proper run types can be found here: [ECSDataAdapters.h](https://github.com/AliceO2Group/AliceO2/blob/dev/DataFormats/Parameters/include/DataFormatsParameters/ECSDataAdapters.h#L54). The `default` can be used 
 to ignore the run or the beam type. 
-The beam type is one of the following: `PROTON-PROTON`, `Pb-Pb`, `Pb-PROTON`, `cosmic`.
+The beam type is one of the following: `pp`, `PbPb`, `pPb`, `cosmic`.
```

Comment:
```suggestion
The beam type comes from the parameter `pdp_beam_type` set by ECS and can be one of the following: `pp`, `PbPb`, `pPb`, `pO`, `OO`, `NeNe`, `cosmic`, `technical`.
See `[readout-dataflow](https://github.com/AliceO2Group/ControlWorkflows/blob/master/workflows/readout-dataflow.yaml)` to verify the possible values.
```
Given that we changed the source of this, we can be more helpful in explaining where the expected values come from.